examples/getting_started/3_Interactivity.ipynb

```diff
@@ -298,7 +298,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Note that if you wish to try the interactivity offered by `%matplotlib widget` you will need to pip or conda install the `ipympl` package."
+    "Note that if you wish to try the interactivity offered by `%matplotlib widget` you will need to pip or conda install the `ipympl` package. If you do not have `ipympl` already, make sure to restart the notebook server after installation before re-executing the notebook."
```

Comment:
Are you saying that if they already had ipympl installed but weren't yet using it, they don't need to restart the server and re-execute? I wouldn't have thought that installing would affect that.

Reply:
Maybe: 

```suggestion
    "Note that if you wish to try the interactivity offered by `%matplotlib widget` you will need to pip or conda install the `ipympl` package, clear the notebook output, restart your notebook server, and re-execute the notebook."
```

Reply:
> Are you saying that if they already had ipympl installed but weren't yet using it, they don't need to restart the server and re-execute?

I'm not sure tbh. I think your suggestion is clearer anyway.